packages/greenfield-app/package.json

```diff
@@ -4,10 +4,11 @@
   "private": true,
   "scripts": {
     "dev": "next dev",
-    "build": "next build",
+    "build": "next build && yarn wagmi generate",
```

Comment:
should we not generate first? in case we use these hooks later for the build?

Reply:
great spot